doc/source/user_guide/indexing.rst

```diff
@@ -1870,25 +1870,34 @@ This is the correct access method:
 
 .. ipython:: python
 
-   dfc = pd.DataFrame({'A': ['aaa', 'bbb', 'ccc'], 'B': [1, 2, 3]})
-   dfc.loc[0, 'A'] = 11
-   dfc
+   dfc = pd.DataFrame({'a': ['one', 'one', 'two',
+                             'three', 'two', 'one', 'six'],
+                       'c': np.arange(7)})
+   dfd = dfc.copy()
+   # Setting multiple items using a mask (recommended)
```

Comment:
you don't need the word recommended here (instead you can use it around L1869)

Reply:
I put it there as @TomAugspurger suggested it in https://github.com/pandas-dev/pandas/issues/34383 if I understood correctly.

Reply:
I think Jeff is saying that "This is the correct access method" implies that this is the recommended way to set things.

Perhaps change "This is the correct access method" to something like "Multiple items can be set with `.loc` and a mask."

Reply:
fine with me.

Reply:
fixed in 5aa53c693

Reply:
please resolve at will